src/SFML/Audio/SoundFileReaderOgg.cpp

```diff
@@ -87,13 +87,6 @@ bool SoundFileReaderOgg::check(InputStream& stream)
 }
 
 
-////////////////////////////////////////////////////////////
-SoundFileReaderOgg::SoundFileReaderOgg()
-{
-    m_vorbis.datasource = nullptr;
```

Comment:
There is no need to set this to `nullptr` because we already initialize `m_vorbis` with `{}` which ensures this particular data member is set to `nullptr`.